plugins/inputs/aos/aos.go

```diff
@@ -103,8 +103,9 @@ func (ssl *streamAos) extractAlertData(alertType string, tags map[string]string,
 		}
 		field_name := propDataType.Prop[i].OrigName
 
-		// Skip field with XXX_
-		if strings.Contains(field_name, "XXX_") {	continue }
+		// Skip field with XXX_ and empty strings
+		if (field_name == "") { continue }
```

Comment:
When can the field_name be empty? Is this indicative of a mismatched proto schema?

Reply:
I don't think this a problem. This issue is most likely a duplicate of #10. Pulling the latest from apstra/telefraf does not exhibit the problem.

Reply:
I'm confused then. Why do we need this change/pr in this case?

Reply:
We don't need this particular change - but I think we should go ahead with the PR for the testing support.

Reply:
Can we revert this change given it's not needed..